### DISCUSSION TITLE
Support for FreshRSS / Google Reader / Fever API

### DISCUSSION THREAD

**Discussion by candidexmedia** [spacecowboy/Feeder#254] [Ideas]
Hi there! Thank you again for creating this app. I'd like to add a feature request for API support, ie Fever API and Google Reader API. It would allow users to manage their feeds in a centralized server, and sync the lists, categories, read status, etc. across different readers and devices.

[Fresh RSS](https://github.com/FreshRSS/FreshRSS) is an example of such server, and it [can sync to a number of apps](hhttps://github.com/FreshRSS/FreshRSS?tab=readme-ov-file#apis--native-apps). I've tried loading the RSS output link generated by FreshRSS into Feeder, but ran into a 404 error.

**Comment by spacecowboy** [comment]
404 is a server error.

but with regards to the actual request, this is quite unlikely today. I built Feeder explicitly to have zero need for a centralized server.

I'm not opposed to syncing subscriptions and read status to some service, but I have no need for it so I won't built it. Someone interested would have to put in the work.
